tests/functional/scripts/run-ft-das.sh

```diff
@@ -72,9 +72,7 @@ python ${UMBRELLA_PATH}/tests/functional/scripts/rstuf-admin-metadata-update.py
   "[select] Select a key for signing (JanisJoplin/JimiHendrix)": "JanisJoplin",
   "(Sign 1) Please enter path to public key": "tests/files/key_storage/JJ.ecdsa",
   "(Sign 1) Please enter password": "hunter2",
-  "[select] Select a key for signing (continue/JimiHendrix)": "JimiHendrix",
-  "(Sign 2) Please enter path to public key": "tests/files/key_storage/JH.ed25519",
-  "(Sign 2) Please enter password": "hunter2"
+  "[select] Select a key for signing (continue/JimiHendrix)": "continue"
```

Comment:
When do you finish the metadata update?

Reply:
`continue`
You gave one signature, and it finished the metadata update process.

![Screenshot 2024-07-30 at 12 46 56](https://github.com/user-attachments/assets/d4991cca-d34c-499b-972b-ace1fadfead6)
![Screenshot 2024-07-30 at 12 47 09](https://github.com/user-attachments/assets/0b6b2aee-22bb-4b98-a5ba-9baf4e9f998a)